test/Libraries/PackageManagerTests/PackageLoaderCustomTest.cs

```diff
@@ -41,19 +41,27 @@ public void LoadFromMultipleLocationTest()
             Assert.AreEqual(loader.DefaultPackagesDirectory,
                 Path.Combine(TestDirectory, "pkgs", "multiple_locations", "folder1", "packages"));
 
-            Assert.AreEqual(2, loader.LocalPackages.Count());
+            var expectedLoadedPackageNum = 0;
+            foreach(var pkg in loader.LocalPackages)
+            {
+                if(pkg.Name == "Custom Rounding" || pkg.Name == "GetHighest")
+                {
+                    expectedLoadedPackageNum++;
+                }
+            }
+            Assert.AreEqual(2, expectedLoadedPackageNum);
```

Comment:
Wouldn't it be easier to just relax the assertion to do `>=` rather than `=`?

Reply:
larger than does not mean the packages will be loaded so I prefer being more specific here.